src/main/java/executor/command/CommandHelp.java

```diff
@@ -4,29 +4,49 @@
 import ui.Ui;
 import ui.Wallet;
 
+import java.security.spec.ECField;
+import java.util.ArrayList;
+import java.util.Collections;
+
 public class CommandHelp extends Command {
 
     /**
      * Constructor to provide the user with the details about the commands available.
      */
     public CommandHelp(String userInput) {
         this.userInput = userInput;
-        this.description = "Provides the user with all the available commands and descriptions.";
+        this.description = "Provides the user with all the available commands and descriptions.\n"
+                + "FORMAT : help \n";
         this.commandType = CommandType.HELP;
     }
 
     @Override
     public void execute(TaskList taskList) {
-        for (String s : CommandType.getNames()) {
-            if (!s.equals("ERROR") && !s.equals("TASK") && !s.equals("BLANK")) {
-                CommandType commandType = CommandType.valueOf(s);
-                Command c = Executor.createCommand(commandType, "null");
-                String commandDesc = c.getDescription();
+          String[] checkerForExtraInput;
+          checkerForExtraInput = userInput.split(" ");
```

Comment:
I think we can use the parser method we already have. So we can like :
_String additionalByUser = _parseForFlag (" help " , userInput);__ 

So whatever that comes after help will be stored in _addtionalByUser_

It will be null if the User enter just _help_ , or else it won't be null. 

So later on in your if else statement, you can check for null and if it is not null, you will print the error message.